src/Campaigns/resources/admin/components/CampaignDetailsPage/Tabs/Settings.tsx

```diff
@@ -111,8 +123,11 @@ export default () => {
 
                         <select {...register('goalType')}>
                             <option value="amount">{__('Amount raised', 'give')}</option>
-                            <option value="donation">{__('Number of Donations', 'give')}</option>
-                            <option value="donors">{__('Number of Donors', 'give')}</option>
+                            <option value="donations">{__('Number of donations', 'give')}</option>
+                            <option value="donors">{__('Number of donors', 'give')}</option>
+                            <option value="amountFromSubscriptions">{__('Recurring amount raised', 'give')}</option>
```

Comment:
These should be displayed only if the recurring add-on is installed.

Reply:
Resolved in 373be0c05a118d99af9801c02e4d66a97e196425